try:
  print(x)
except:
  print("An exception occurred, x is not defined")